recipes/f2c/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copy Makefile from the source directory
+
+cp ${RECIPE_DIR}/Makefile .
```

Comment:
Why? Isn't the Makefile from the source archive enough?

Reply:
The makefile in the package does not copy over header files, which are needed by some program to compile against f2c. 

However, following your suggestion, I removed that Makefile. I now copy the one missing header file manually in build.sh.